src/com/booksaw/betterTeams/Team.java

```diff
@@ -39,6 +35,7 @@
  * @author booksaw
  *
  */
+@SuppressWarnings("SameParameterValue")
```

Comment:
What is causing this warning? I think it may be wiser to suppress the specific methods which are causing problems instead of suppressing the warning for the entire class.